scipy/stats/_continuous_distns.py

```diff
@@ -8957,8 +8967,11 @@ def _stats(self, chi):
         chi2 = chi**2
         phi = _argus_phi(chi)
         m = np.sqrt(np.pi/8) * chi * sc.ive(1, chi2/4) / phi
-        v = (1 - 3 / chi2 + chi * _norm_pdf(chi) / phi) - m**2
-        return m, v, None, None
+        # compute second moment, use Taylor expansion for small chi
+        mu2 = _lazywhere(chi > 0.01, (chi,),
+            lambda c: 1 - 3 / c**2 + c * _norm_pdf(c) / _argus_phi(c),
```

Comment:
i did not change this one since to allow for proper broadcasting, I assume we need to rely on _argus_phi

Reply:
Yeah, I see. Well, I'll leave it up to you. Depends on how expensive `_argus_phi` is. 
You could leave it alone, or you could use the mask `chi > 0.01` to index `phi` yourself, something like:
```python3
mu2 = np.empty_like(chi)
mask = chi > 0.01
c = chi[mask]
mu2[mask] = 1 - 3 / c**2 + c * _norm_pdf(c) / phi[mask]
c= chi[~mask]
mu2[~mask] = np.polyval([2./2625, 0, 6./175, 0, 0.4], c))
```
(or, it would probably be faster to just start with `mu2 = 1 - 3 / chi**2 + chi * _norm_pdf(chi) / phi` and replace the `~mask` values)

Reply:
Just let me know what you think of this. I can close it if you prefer.

Reply:
good point, I will integrate this into the code

Reply:
Did you decided against it after all?

Reply:
For some annoying reason, it did not work and I haven't yet figured out why. The tests that check the variance for large chi fail if I use the code above. It looks like the assignment `mu2[mask] = ...` does not update mu2 correctly (mu2 was zero when I tried), I need to check again.